### DISCUSSION TITLE
Is there a way to get keys of a map and values of a map as an array

### DISCUSSION THREAD

**Discussion by ashish01** [duckdb/duckdb#6046] [Q&A]
e.g. `map_keys(map([1, 2], ['a', 'b']))` returns `[1, 2]` and `map_values(map([1, 2], ['a', 'b']))` returns `['a', 'b']`

**Comment by ashish01** [comment]
Looks like @Tishj already created a pull request adding this function. Was it merged?

**Reply by Tishj** [reply]
Hey yes I added that PR but it was missing some coverage, and the internals of the map have been changed in the meantime, so it kinda stranded.

They are very easy functions though, especially now that MAP is physically a `LIST(STRUCT{key,value})` now
Previously this was `STRUCT{LIST(key), LIST(value)}`

Definitely a good first issue if you want to implement them :)
